Source/Logs.swift

```diff
@@ -8,22 +8,23 @@ class Logs {
     enum Level {
         case debug, info, warn, error
 
-        #if !os(Linux)
+#if !os(Linux)
```

Comment:
The changes in this file are a fix to a change introduced in a recent unreleased PR that broke compatibility with Swift <5.4 on Linux. Prior to 5.4, `String` does not comply to `CVarArg`. The dirty solution here is just to have the type of `log` be different on Linux since the values aren't used anyway.